composer.json

```diff
@@ -41,6 +42,25 @@
         "psr-4": {
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "PhpList\\BaseDistribution\\Tests\\": "Tests/"
+        }
+    },
+    "scripts": {
```

Comment:
This script copies only the web/ directory of the core package into the "distribution" application, not the complete core package.

This is a long-term thing.

We need to have a document root outside of the updatable packages (i.e., outside of vendor/) so we can put stuff there from multiple packages (e.g., assets from the web-frontent package) and also add stuff manually without the directory getting reset to a pristine state when the dependencies get updated.